recipes/toolchain/deactivate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
> Strictly speaking, I don't think we really need to deactivate.

So, I'm thinking about this from a versatility perspective. If one wants to use this locally, to build something and then switch environments. We should really try to clean stuff up so they don't have a surprise error down the line.

> It isn't as if we are putting the environment back into the state we started with anyway (some of these may have been defined before activation).

Eek. 🐭 I hope not. Personally, I try to keep my environment super pristine. There are just way too many things environment variables affect. Besides, as an end user, I think it is much easier to solve this scenario, "Oh, apparently this isn't set. Let's set it.", than this one, "It does appear to be set. What is wrong with it?". In the future, we certainly can discuss storing the pre-activation environment variables somewhere else and then reloading them during post-activation.